paper-toolbar.html

```diff
@@ -106,14 +106,17 @@
   <template>
     <style>
       :host {
+        --calculated-paper-toolbar-height: var(--paper-toolbar-height, 64px);
+        --calculated-paper-sm-toolbar-height: var(--paper-toolbar-sm-height, 56px);
```

Comment:
`--calculated-paper-sm-toolbar-height` -> `--calculated-paper-toolbar-sm-height`, to match the variable name.